pkg/chunk/aws/dynamodb_storage_client.go

```diff
@@ -749,6 +753,10 @@ func (b dynamoDBWriteBatch) Add(tableName, hashValue string, rangeValue []byte,
 	})
 }
 
+func (b dynamoDBWriteBatch) Delete(tableName, hashValue string, rangeValue []byte) {
+	panic("implement me")
```

Comment:
Instead of implement me can you make this panic be a bit more informative

Reply:
Sure will do. Thanks!